File: app/src/main/java/me/devsaki/hentoid/parsers/DummyParser.java
package me.devsaki.hentoid.parsers;

import java.util.List;

import me.devsaki.hentoid.database.domains.Content;
import me.devsaki.hentoid.database.domains.ImageFile;

public class DummyParser implements ImageListParser {
    @Override
    public List<ImageFile> parseImageList(Content content) {
        return content.getImageFiles();
    }

    @Override
    public ImageFile parseBackupUrl(String url, int order) {
        return ParseHelper.urlToImageFile(url, order);
    }
}

File: app/src/main/java/me/devsaki/hentoid/parsers/images/DummyParser.kt
package me.devsaki.hentoid.parsers.images

import me.devsaki.hentoid.database.domains.Content

class DummyParser : BaseImageListParser() {
    override fun parseImages(content: Content): List<String> {
        return content.imageList.map { img -> img.url }
    }
}